tests/main.test.js

```diff
@@ -152,3 +152,23 @@ test("deliberately skipped .css shouldn't error", async () => {
   })
   expect(finalCss).toEqual('p{color:brown}')
 })
+
+test.skip('leaves used css variables', async () => {
```

Comment:
Can you add a comment above this one *why* this test is currently skipped. 
Perhaps a comment that contains a URL to a filed issue about it.